.github/workflows/update_documentation.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v4
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7
+          ruby-version: 3.4
```

Comment:
What's this version bump for?

Reply:
I see it breaks down markdownlint: https://github.com/rucio/documentation/actions/runs/14323454114/job/40144609662?pr=540

LGTM, ideally should have been separated in a separate commit though